daemon/internal/newrelic/integration/test.go

```diff
@@ -200,35 +200,39 @@ func merge(a, b map[string]string) map[string]string {
 }
 
 func (t *Test) MakeRun(ctx *Context) (Tx, error) {
-	env := merge(ctx.Env, t.Env)
+	for k, v := range ctx.Env {
```

Comment:
The `merge()` call in the previous code would give values in `t.Env` precedence over values in `ctx.Env`.  This new code does the opposite.
My suggestion would be to call `merge()` as before and store the result in `t.Env` to ensure  operation is the same as previously.

Reply:
@mfulb How about 5064b7c93267fae793fbef33760b1df37fc35d77?